statemachine/signature.py

```diff
@@ -1,4 +1,5 @@
 import itertools
+from asyncio import iscoroutinefunction
```

Comment:
`iscoroutinefunction` is from "inspect" (`from inspect import iscoroutinefunction`) - asyncio is certainly just importing it from there, and making use of it.

Reply:
Oh, thanks! Found it: https://docs.python.org/3/library/inspect.html#inspect.iscoroutinefunction